utils/llm.py

```diff
@@ -27,8 +27,7 @@
             "30": 0.002,
             "50": 0.004,
             "100": 0.007,
-            # NOTE: Why is this one a string?
-            "150": "0.01"}
```

Comment:
Mistake on my part probably. I really like beartype btw. It might be very useful to quickly debug those in the whole code if you want.